lib/logstash/codecs/identity_map_codec.rb

```diff
@@ -0,0 +1,255 @@
+# encoding: utf-8
+require "logstash/namespace"
+# require "concurrent"
+require "thread_safe"
```

Comment:
sure which is why there is a hint to require `concurrent`. I half expected the concurrent v1.0 release while this code change was still unmerged.